pkg/concurrency/value_stream_bench_test.go

```diff
@@ -223,33 +209,27 @@ func BenchmarkUnbufChanRead(b *testing.B) {
 }
 
 func BenchmarkSliceRead(b *testing.B) {
-	b.StopTimer()
 
 	slice := make([]int, 0, b.N)
-	for i := 0; i < b.N; i++ {
+	for i := 0; b.Loop(); i++ {
 		slice = append(slice, i)
 	}
 
-	b.StartTimer()
-
-	for i := 0; i < b.N; i++ {
+	for b.Loop() {
 		slice = slice[1:]
 	}
 	require.Empty(b, slice)
 }
 
 func BenchmarkSliceReadWithMutex(b *testing.B) {
-	b.StopTimer()
 
 	slice := make([]int, 0, b.N)
-	for i := 0; i < b.N; i++ {
+	for i := 0; b.Loop(); i++ {
```

Comment:
I think, this is benchmark setup and it should not be benchmarked.

It fails currently with:

```
BenchmarkSliceRead
    benchmark.go:417: B.Loop called with timer stopped
--- FAIL: BenchmarkSliceRead
BenchmarkSliceReadWithMutex
    benchmark.go:417: B.Loop called with timer stopped
--- FAIL: BenchmarkSliceReadWithMutex
FAIL
exit status 1
FAIL	github.com/stackrox/rox/pkg/concurrency	0.011s
```